### DISCUSSION TITLE
Add `one tsc verify` or similar

### DISCUSSION THREAD

**Discussion by alecmev** [paularmstrong/onerepo#685] [Ideas]
`useProjectReferences` is great, but we also want to check if everything is correct in a lint task and in CI, and currently there's no way to do that without potentially writing to disk.

Alternatively, instead of `tsc verify`, an option allowing the plugin to write could be added (defaulting to `false`).

**Comment by paularmstrong** [comment]
We might want to think about this a little more holistically across commands that do similar things. A standard flag may be better, eg `--no-fix`, analogous to the opposite of eslint's `--fix` or a global `--ci` flag. Prettier also has a `--check` flag. It would be good to make the flags the same across all commands to avoid confusion.
